packages/apps/explorer/src/services/format.ts

```diff
@@ -1,13 +1,21 @@
 import type { NetworkInfo } from '@/__generated__/sdk';
 
-export function formatNumberWithUnit(number: number, unit?: string): string {
+/**
+ * this function gets your numeric value and returns a string with the value and the unit (e.g. 1023 will be formatted to 1.23 K)
+ * @param number Value to be formatted
+ * @returns string value with unit
+ *
+ */
+export function formatNumberWithUnit(number: number): string {
   if (number === 0) {
-    return `0 ${unit !== undefined ? unit : ''}`;
+    return `0`;
```

Comment:
it returns undefineds again
![image](https://github.com/kadena-community/kadena.js/assets/4015521/71387214-342b-4bd4-9a95-27fb94defe8e)

Reply:
Good catch! The issue here was with the unit. Fixed it